openthread_border_router/Dockerfile

```diff
@@ -79,6 +82,9 @@ RUN \
         -DOTBR_NAT64=ON \
         -DOT_POSIX_NAT64_CIDR="192.168.255.0/24" \
         -DOTBR_DNS_UPSTREAM_QUERY=ON \
+        -DOT_CHANNEL_MONITOR=ON \
+        -DOT_COAP=OFF \
+        -DOT_COAPS=OFF \
```

Comment:
this wasn't mentioned in the PR description, what does it do ?

Reply:
Added a note! :+1: